### ISSUE TITLE
vista#sidebar#Toggle as command

### ISSUE THREAD

**Issue by Mte90** [liuchengxu/vista.vim#369]
**Is your feature request related to a problem? Please describe.**
I was using Tagbar that has a command that when invoked check if is already opened and close it and viceversa. I see `vista#sidebar#Toggle` method so maybe is just enough to wrap it to pass the executive.

**Comment by liuchengxu** [comment]
You can use `:Vista!!` to toggle the vista window.

**Comment by Mte90** [comment]
Let just to close the window but not also to open it.

**Comment by liuchengxu** [comment]
Just define a command then:

```vim
command! VistaClose call vista#sidebar#Close() 
```

**Comment by Mte90** [comment]
This command just close the panel.
I am a looking for a toggle command, if the panel is closed open it, if open closed it.

**Comment by liuchengxu** [comment]
>I am a looking for a toggle command, if the panel is closed open it, if open closed it.

`:Vista!!`

**Comment by Mte90** [comment]
In my neovim is just closing the panel.
As I open the panel with `Vista nvim_lsp` and `Vista!! nvim_lsp` generate an error.

**Comment by liuchengxu** [comment]
`:Vista!!` accepts no args, you just call `:Vista!!` to toggle the vista panel.

**Comment by Mte90** [comment]
If I run `:Vista!!` it isn't opening, it lets me to just close an already opened panel.

Reading the docs I need to open the panel to specify the service or doesn't open so on my tests I did also for that command too.

**Comment by liuchengxu** [comment]
Can not reproduce, please file a new issue with detailed reproduction info.

**Comment by Mte90** [comment]
My dotfiles are https://github.com/mte90/dotfiles and my vista settings are https://github.com/mte90/dotfiles/blob/master/.vim/custom/custom-vista.vim

I think that the opening with that command doesn't work because the first service is ale and not nvim_lsp as if I run Vista without anything is still empty.

Reading on issues or the code I don't remember I saw that in case it is empty or there is an error the panel is not showed but there is no error about it. With Vista nvim_lsp I was able to open the panel with no troubles.
